device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D44C10/pg_profile_lookup.ini

```diff
@@ -0,0 +1,38 @@
+##
```

Comment:
Let's make this a symbol link to `../Mellanox-SN2700-D40C8S8/pg_profile_lookup.ini`